google/cloud/forseti/notifier/notifier.py

```diff
@@ -74,58 +74,60 @@ def convert_to_timestamp(violations):
     return violations
 
 
-def process(message):
-    """Process messages about what notifications to send.
+def run_inv_summary(inv_index_id, service_config):
+    """Emit an inventory summary notification if/as needed.
 
     Args:
-        message (dict): Message with payload in dict.
-            The payload will be different depending on the sender
-            of the message.
-
-            Example:
-                {'status': 'foobar_done',
-                 'payload': {}}
+        inv_index_id (str): Inventory index id.
+        service_config (ServiceConfig): Forseti 2.0 service configs.
     """
-    payload = message.get('payload')
-
-    if message.get('status') == 'inventory_done':
-        inv_email_notifier = inv_summary.EmailInventorySnapshotSummary(
-            payload.get('sendgrid_api_key')
-        )
-        inv_email_notifier.run(
-            payload.get('cycle_time'),
-            payload.get('cycle_timestamp'),
-            payload.get('snapshot_cycle_status'),
-            payload.get('notifiers'),
-            payload.get('email_sender'),
-            payload.get('email_recipient')
-        )
+    notifier_config = service_config.get_notifier_config()
+    if not notifier_config.get('inventory'):
+        LOGGER.info('No "inventory" configuration for notifier.')
+        return
+
+    if not notifier_config.get('inventory').get('summary'):
+        LOGGER.info('No "inventory summary" configuration for notifier.')
+        return
+
+    notifier_config = notifier_config.get('inventory').get('summary')
```

Comment:
Instead of `notifier_config`, this should be called `inv_sumary_config`.  This way, it will be much more clear where it's used below.

Reply:
Done.